tests/test_strongly_connected.py

```diff
@@ -0,0 +1,63 @@
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+import unittest
+
+import retworkx
+
+
+class TestStronglyConnected(unittest.TestCase):
+
+    def test_number_strongly_connected_all_strong(self):
+        G = retworkx.PyDiGraph()
+        node_a = G.add_node(1)
```

Comment:
It's confusing that this says `add_node(1)` but then it appears as `[0]` below. Is that right?

Reply:
It's because 0 is the node id/index while the weight/data payload is 1. The new function is returning the indexes not the data. I can change the assert to be:

```python3
self.assertEqual(retworkx.strongly_connected_components(G),
                         [[node_c], [node_b], [node_a]])
```

if you think that'll be clearer.

Reply:
Done in: https://github.com/Qiskit/retworkx/pull/111/commits/092badfae7a66f8f400bc62bc3930d55474b3e3e